src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -228,8 +240,113 @@ ucs_status_t uct_rc_mlx5_ep_get_zcopy(uct_ep_h tl_ep, const uct_iov_t *iov, size
     return status;
 }
 
-ucs_status_t uct_rc_mlx5_ep_am_short(uct_ep_h tl_ep, uint8_t id, uint64_t hdr,
-                                     const void *payload, unsigned length)
+#if HAVE_IBV_EXP_DM
+static size_t uct_rc_mlx5_ep_am_short_dm_pack(void *dest, void *arg)
+{
+    uct_rc_mlx5_dm_pack_data_t *data = (uct_rc_mlx5_dm_pack_data_t*)arg;
+    uint64_t *dst                    = dest;
+
+    *dst = data->hdr;
+    memcpy(dst + 1, data->super.payload, data->super.length);
+    return sizeof(data->hdr) + data->super.length;
+}
+
+/* DM memory should be written by 8 bytes (int64) to eliminate
+ * processor cache issues. to make this used uct_rc_mlx5_dm_copy_data_t
+ * structure where first hdr_len bytes are filled by message header
```

Comment:
can you please rephrase it somehow? Upper case after `.`, `uct_rc_mlx5_dm_copy_data_t` is a union, etc

Reply:
updated